docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: "3.5"
+services:
+
+  markbuilder:
+    image: golang:latest
+    environment:
+      # MacOS 64-bit
+      - GOOS=darwin
```

Comment:
Do we really need this as default behavior? Can't we just comment all the entries?

Reply:
Whatever you prefer. These are my settings and it's not an issue at all to change them.

Reply:
I just changed it so that the values are taken from the environment.